src/simplewallet/simplewallet.cpp

```diff
@@ -1362,26 +1480,28 @@ bool simple_wallet::export_multisig_main(const std::vector<std::string> &args, b
 bool simple_wallet::import_multisig(const std::vector<std::string> &args)
 {
   CHECK_MULTISIG_ENABLED();
-  import_multisig_main(args, false);
+  import_multisig_main(args);
   return true;
 }
 
-bool simple_wallet::import_multisig_main(const std::vector<std::string> &args, bool called_by_mms)
+bool simple_wallet::import_multisig_main(const std::vector<std::string> &args)
 {
   CHECK_MULTISIG_ENABLED();
-  const multisig::multisig_account_status ms_status{m_wallet->get_multisig_status()};
+  const MultisigState ms_status{m_wallet_impl->multisig()};
```

Comment:
I wonder whether it would be worth it, and easy enough, to put all this repeated getting multisig state and then checking whether A) is key on device, B) is multisig, C) is ready into its own method ...